### DISCUSSION TITLE
Code issue UniFi protect HA addon

### DISCUSSION THREAD

**Discussion by xeno111** [uilibs/uiprotect#325] [Q&A]
I’m using the UniFi protect addon for Home Assistant and I have a g4 doorbell pro Poe and I’m trying to get a fingerprint automation to work with the supplied documentation but the section for the condition template doesn’t function, if I disable the condition in the automation it runs fine. 
```yaml
condition:
  - condition: template
    value_template: >
      {{
         not trigger.event.data.old_state.attributes.get('restored', false) and
         not trigger.event.data.old_state.state == 'unavailable' and
         trigger.event.data.new_state is not none and
         trigger.event.data.new_state.attributes.event_type == 'identified' and
         (trigger.event.data.new_state.attributes.ulp_id|default('')) != '' and
         trigger.event.data.new_state.attributes.ulp_id in ['ALLOWED_ID1', 'ALLOWED_ID2']
       }}
```
I’m at a loss and I’m not a programmer so don’t understand the code, I’m sure it’s something I’ve done wrong, maybe a typo I don’t know, could you help please?

**Comment by TheJulianJES** [comment]
Did you replace the `ALLOWED_ID1` with an allowed fingerprint ID?

**Comment by xeno111** [comment]
I didn’t, that’s what it is how do I find the fingerprint id?On 14 Dec 2024, at 22:18, TheJulianJES ***@***.***> wrote:﻿
Did you replace the ALLOWED_ID1 with an allowed fingerprint ID?

**Comment by xeno111** [comment]
I found the id’s in protect and added them and I can confirm it works flawlessly! Thank you for fast response and pointing me in the right direction, keep up your excellent work, thank you
